packages/next-elba/config.d.ts

```diff
@@ -0,0 +1,24 @@
+import { type User } from '@elba-security/sdk';
+import { type Organisation, type TableColumns } from './database/schema';
+
+// eslint-disable-next-line @typescript-eslint/ban-types -- don't care
+export type ElbaConfig<T extends TableColumns = {}> = {
+  name: string;
+  database: {
+    organisations: T;
```

Comment:
```suggestion
    organisations: T;
    encryptedKeys?: (keyof T)[]
```

We could add `encryptedKeys` and automatically encrypt/decrypt keys from db.